homeassistant/components/signal_messenger/notify.py

```diff
@@ -100,10 +114,14 @@ def send_message(self, message: str = "", **kwargs: Any) -> None:
         attachments_as_bytes = self.get_attachments_as_bytes(
             data, CONF_MAX_ALLOWED_DOWNLOAD_SIZE_BYTES, self._hass
         )
-
+        text_mode = self.get_text_mode(data)
         try:
             self._signal_cli_rest_api.send_message(
-                message, self._recp_nrs, filenames, attachments_as_bytes
+                message,
+                self._recp_nrs,
+                filenames,
+                attachments_as_bytes,
+                text_mode=text_mode,
```

Comment:
```suggestion
                text_mode=kwargs.get(ATTR_TEXTMODE),
```

Reply:
This had to be changed to `text_mode="normal" if data is None else data.get(ATTR_TEXTMODE)` to account for `data = None`.